digitaltwin/modelrepo/AuthorizationDelegatingHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Azure.Devices.Common.Authorization;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.DigitalTwin.Model.Service
+{
+    internal class AuthorizationDelegatingHandler : DelegatingHandler
+    {
+        IoTServiceClientCredentials ioTServiceClientCredentials;
+        public AuthorizationDelegatingHandler(IoTServiceClientCredentials ioTServiceClientCredentials)
+        {
+            this.ioTServiceClientCredentials = ioTServiceClientCredentials;
```

Comment:
now that the name is disambiguated with the underscore prefix, you can remove "this."